kbase/narrative
# DATAUP-234 remove old integration test cruft

# Description of PR purpose/changes

DO NOT MERGE before merging #1863 , as this removes the CI testing stuff, and expects the github actions to be present.

This removes code and references to older selenium code that's no longer used. It also removes some references to PhantomJS, which is also no longer used.

This is semi-dependent on #1863 and should be rebased if that's accepted first.

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X

# Testing Instructions
* Details for how to test the PR: 

1. Tests shouldn't run in Travis
2. Tests should run and pass in Github actions.

- [ ] Tests pass in github actions and locally
